types/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@joystream/types",
-  "version": "2.1.0",
+  "version": "2.3.0",
```

I know I already suggested 2.3.0, but seeing that nara will also need a version bump. Lets update major version instead (I think this is what we came to as best approach with Leszek)

```suggestion
  "version": "4.0.0",
```

And Nara will be version `3.0.0`